src/modules/collection/selectors.ts

```diff
@@ -105,10 +108,22 @@ export const getStatusByCollectionId = createSelector<
         }
       }
     }
-    return statusByCollectionId
+    return statusByCollectionId[collectionId]
   }
 )
 
+export const isThirdPartyCollection = createSelector<
```

Comment:
I thought it'd be nice to have a selector that memoizes the validation if a collection is a TP based on a specific `collectionId`.

Reply:
This seems cool! I'm not entirely sure how reselect handles these cases tho, part of me thinks that's not memoizing but I'm not sure

@cazala @LautaroPetaccio halp

Reply:
Nevermind guys 😛 , I wrote that selector when I thought the only way to know if an item was a TPW was through the collection so now that I'm using the `item.urn`, this is not longer needed.

To Juan's question tho, I was thinking of using the selector concatenated with other selectors in the `getStatusByItemId` one. Something like:
```
  state => getItems(state),
  state => getEntityByItemId(state),
  state => getCurationsByCollectionId(state),
  (_state, collectionId) => isThirdPartyCollection(collectionId),
  getItemCurationsByItemId,
```